### PR TITLE
distro fix + fixes runtimes

### LINKED ISSUE #11635: Atmos distro loop still messed up
Apparently @duncathan missed a spot with #11604. Atmos works for the most part but the distro loop still starts out with a strange 50/50 Oxygen and N2 mix at something around 600 kPa.

### PR DESCRIPTION
**Pull request by duncathan**
Fixes #11635 
I like the idea that before this fix an air canister was basically just someone pumping an oxygen canister into an n2 canister then painting it white

this also fixes multiple runtimes

### CODE REVIEW

**Comment by duncathan** [comment]
just fixed all the atmos-related runtimes I found in the past couple of days

**Comment by Aranclanos** [comment]
travis said no

File: code/ATMOSPHERICS/pipes/heat_exchange/he_pipes.dm [hunk 1]
```diff
@@ -49,13 +49,11 @@
...
 
+
 /obj/machinery/atmospherics/pipe/heat_exchanging/process()
 	var/datum/gas_mixture/pipe_air = return_air()
-	//Burn any mobs buckled to ourselves based on our temperature
-	if(buckled_mob)
-		var/heat_limit = 1000
-		if(pipe_air.temperature > heat_limit + 1)
-			buckled_mob.apply_damage(4 * log(pipe_air.temperature - heat_limit), BURN, "chest")
+	if(!return_air)
```
Comment (kyrahabattoir) [inline code/ATMOSPHERICS/pipes/heat_exchange/he_pipes.dm]: Shouldn't this be a method call rather than a variable?

Comment (duncathan) [inline code/ATMOSPHERICS/pipes/heat_exchange/he_pipes.dm]: Lololol

Comment (RemieRichards) [inline code/ATMOSPHERICS/pipes/heat_exchange/he_pipes.dm]: there's already a call for it a line up, just reuse the ```pipe_air``` var.

Comment (duncathan) [inline code/ATMOSPHERICS/pipes/heat_exchange/he_pipes.dm]: Yeah that's what I'd meant to use woops

**Pull request merged**